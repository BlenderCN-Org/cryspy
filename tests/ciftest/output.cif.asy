// Crystal Structure Drawing Program DRAWxtl V5.5

// Copyright 1996, 1997, 2000-2011 by Larry Finger, Martin Kroeker, and Brian Toby
//   (all rights reserved).

// Scene file created from output.cif.str 
// with command line options: -b  11.14  25.97  12.10 -o  0.50  0.50  0.50 
// -p -0.05  1.05 -0.05  1.05 -0.05  1.05 -v 14.30 -23.90 -36.10 -m  1.00


import three;
unitsize(0.1cm);
size(250,250);
viewportmargin=(1cm,1cm);
currentprojection=perspective ( 0.00, 0.00,406.81,target=( 0.00, 0.00,0.),up=Y);

picture pic;
transform3 t=rotate(14.30,X)*rotate(-23.90,Y)*rotate(-36.10,Z);

 draw(pic, (-3.71440,-8.65655,-4.03480)--(-3.71440,-8.65655, 4.03480)--(-3.71440, 8.65655, 4.03480)--(-3.71440, 8.65655,-4.03480)--cycle,rgb(0.00,0.00,0.00) );
 draw(pic, ( 3.71440,-8.65655,-4.03480)--( 3.71440, 8.65655,-4.03480)--( 3.71440, 8.65655, 4.03480)--( 3.71440,-8.65655, 4.03480)--cycle,rgb(0.00,0.00,0.00) );
 draw(pic, (-3.71440,-8.65655,-4.03480)--( 3.71440,-8.65655,-4.03480),rgb(0.00,0.00,0.00) );
 draw(pic, (-3.71440,-8.65655, 4.03480)--( 3.71440,-8.65655, 4.03480),rgb(0.00,0.00,0.00) );
 draw(pic, (-3.71440, 8.65655, 4.03480)--( 3.71440, 8.65655, 4.03480),rgb(0.00,0.00,0.00) );
 draw(pic, (-3.71440, 8.65655,-4.03480)--( 3.71440, 8.65655,-4.03480),rgb(0.00,0.00,0.00) );
 // Sphere for Dy_  at  0.70700  0.32800  0.05200  
draw(pic, shift ( 1.53776, -2.97785, -3.61518)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.98200  0.08200  0.00000  
draw(pic, shift ( 3.58068, -7.23688, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at -0.01800  0.08200  0.00000  
draw(pic, shift (-3.84812, -7.23688, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at -0.01800  0.08200  1.00000  
draw(pic, shift (-3.84812, -7.23688,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.98200  0.08200  1.00000  
draw(pic, shift ( 3.58068, -7.23688,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.01800  0.91800  1.00000  
draw(pic, shift (-3.58068,  7.23688,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.01800  0.91800  0.00000  
draw(pic, shift (-3.58068,  7.23688, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  1.01800  0.91800  0.00000  
draw(pic, shift ( 3.84812,  7.23688, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  1.01800  0.91800  1.00000  
draw(pic, shift ( 3.84812,  7.23688,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.29300  0.67200  0.94800  
draw(pic, shift (-1.53776,  2.97785,  3.61518)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.00000  0.00000  0.00000  
draw(pic, shift (-3.71440, -8.65655, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.00000  0.00000  1.00000  
draw(pic, shift (-3.71440, -8.65655,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.00000  1.00000  0.00000  
draw(pic, shift (-3.71440,  8.65655, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.00000  1.00000  1.00000  
draw(pic, shift (-3.71440,  8.65655,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  1.00000  0.00000  0.00000  
draw(pic, shift ( 3.71440, -8.65655, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  1.00000  0.00000  1.00000  
draw(pic, shift ( 3.71440, -8.65655,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  1.00000  1.00000  0.00000  
draw(pic, shift ( 3.71440,  8.65655, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  1.00000  1.00000  1.00000  
draw(pic, shift ( 3.71440,  8.65655,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.10800  0.47100  0.00000  
draw(pic, shift (-2.91209, -0.50208, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.10800  0.47100  1.00000  
draw(pic, shift (-2.91209, -0.50208,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.89200  0.52900  1.00000  
draw(pic, shift ( 2.91209,  0.50208,  4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 // Sphere for Dy_  at  0.89200  0.52900  0.00000  
draw(pic, shift ( 2.91209,  0.50208, -4.03480)*scale3(0.40)*unitsphere,rgb(1.00,0.00,1.00));
 label(pic, "o",(-4.21213,-9.15863,-4.53512));
 label(pic, "a",( 4.21213,-8.65655,-4.03480));
 label(pic, "b",(-3.71440, 9.15863,-4.03480));
 label(pic, "c",(-3.71440,-8.65655, 4.53512));
add(t*pic);
