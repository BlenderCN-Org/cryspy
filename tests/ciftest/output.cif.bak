data_global _chemical_name 'New material'
_cell_length_a 1.0
_cell_length_b 1.0
_cell_length_c 1.0
_cell_angle_alpha 90
_cell_angle_beta 90
_cell_angle_gamma 90
_symmetry_space_group_name_H-M 'P 1'


loop_
 _atom_site_label 
 _atom_site_type_symbol 
 _atom_site_fract_x 
 _atom_site_fract_y 
 _atom_site_fract_z 
  Dy1_1 O 0.7070(10) 0.3280(10) 0.0520(10)
  Dy1_2 Dy 0.982(0) 0.082(0) 1/4
  Dy1_3 Dy 0.018000000000000016(0) 0.918(0) 3/4
  Dy1_4 O 0.2930(10) 0.6720(10) 0.9480(10)
  Dy1_5 Mn 1/2 0 0
  Dy1_6 O 0.1080(20) 0.4710(20) 1/4
  Dy1_7 O 0.8920(20) 0.5290(20) 3/4
  Mn1_1 O 0.7070(10) 0.3280(10) 0.0520(10)
  Mn1_2 Dy 0.982(0) 0.082(0) 1/4
  Mn1_3 Dy 0.018000000000000016(0) 0.918(0) 3/4
  Mn1_4 O 0.2930(10) 0.6720(10) 0.9480(10)
  Mn1_5 Mn 1/2 0 0
  Mn1_6 O 0.1080(20) 0.4710(20) 1/4
  Mn1_7 O 0.8920(20) 0.5290(20) 3/4
  O1_1 O 0.7070(10) 0.3280(10) 0.0520(10)
  O1_2 Dy 0.982(0) 0.082(0) 1/4
  O1_3 Dy 0.018000000000000016(0) 0.918(0) 3/4
  O1_4 O 0.2930(10) 0.6720(10) 0.9480(10)
  O1_5 Mn 1/2 0 0
  O1_6 O 0.1080(20) 0.4710(20) 1/4
  O1_7 O 0.8920(20) 0.5290(20) 3/4
  O2_1 O 0.7070(10) 0.3280(10) 0.0520(10)
  O2_2 Dy 0.982(0) 0.082(0) 1/4
  O2_3 Dy 0.018000000000000016(0) 0.918(0) 3/4
  O2_4 O 0.2930(10) 0.6720(10) 0.9480(10)
  O2_5 Mn 1/2 0 0
  O2_6 O 0.1080(20) 0.4710(20) 1/4
  O2_7 O 0.8920(20) 0.5290(20) 3/4

