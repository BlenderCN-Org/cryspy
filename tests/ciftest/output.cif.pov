/*

POVxtl-Persistence of Vision Ray Tracer
Crystal Structure Drawing Program V5.5

Copyright 1996, 1997, 2000-2011 by Larry Finger, Martin Kroeker, and Brian Toby
   (all rights reserved).

Scene file created from output.cif.str 
with command line options: -b  11.14  25.97  12.10 -o  0.50  0.50  0.50 
-p -0.05  1.05 -0.05  1.05 -0.05  1.05 -v 14.30 -23.90 -36.10 -m  1.00
*/

#if (version>3.65)
#version 3.6;
#end

#include "colors.inc"
#include "chars.inc"

default {
        finish {
                ambient  0.70
                diffuse  0.30
                specular  0.08
                roughness  0.01
               }
        }

camera{
 location< 0.000000 ,0.000000, 406.814362>
 up <0, 0.1, 0>
 right <-0.1, 0, 0>
 look_at <0.000000,0.000000,0>
}
background{color White}
light_source {< 0, 0, 254.258972>
color red 4.0 green 4.0 blue 4.0
}
object{
 union{

#declare xrot =  14.3;
#declare yrot = -23.9;
#declare zrot = -36.1;
#declare xmove =   0.0;
#declare ymove =   0.0;

  cylinder{<-3.714400,-8.656550,-4.034800>,<-3.714400,-8.656550,4.034800>,0.050852 
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<-3.714400,-8.656550,4.034800>,<-3.714400,8.656550,4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<-3.714400,8.656550,4.034800>,<-3.714400,8.656550,-4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<-3.714400,8.656550,-4.034800>,<-3.714400,-8.656550,-4.034800>, 0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<-3.714400,-8.656550,-4.034800>,<3.714400,-8.656550,-4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<3.714400,-8.656550,-4.034800>,<3.714400,8.656550,-4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<3.714400,8.656550,-4.034800>,<3.714400,8.656550,4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<3.714400,8.656550,4.034800>,<3.714400,-8.656550,4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<3.714400,-8.656550,4.034800>,<3.714400,-8.656550,-4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<3.714400,8.656550,-4.034800>,<-3.714400,8.656550,-4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<-3.714400,8.656550,4.034800>,<3.714400,8.656550,4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
  cylinder{<3.714400,-8.656550,4.034800>,<-3.714400,-8.656550,4.034800>,0.050852
   texture{pigment{color Black }}
   no_shadow no_reflection
  }
 /* Sphere for Dy_  at  0.70700  0.32800  0.05200 */ 
 sphere{< 1.53776, -2.97785, -3.61518>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.98200  0.08200  0.00000 */ 
 sphere{< 3.58068, -7.23688, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at -0.01800  0.08200  0.00000 */ 
 sphere{<-3.84812, -7.23688, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at -0.01800  0.08200  1.00000 */ 
 sphere{<-3.84812, -7.23688,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.98200  0.08200  1.00000 */ 
 sphere{< 3.58068, -7.23688,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.01800  0.91800  1.00000 */ 
 sphere{<-3.58068,  7.23688,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.01800  0.91800  0.00000 */ 
 sphere{<-3.58068,  7.23688, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  1.01800  0.91800  0.00000 */ 
 sphere{< 3.84812,  7.23688, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  1.01800  0.91800  1.00000 */ 
 sphere{< 3.84812,  7.23688,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.29300  0.67200  0.94800 */ 
 sphere{<-1.53776,  2.97785,  3.61518>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.00000  0.00000  0.00000 */ 
 sphere{<-3.71440, -8.65655, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.00000  0.00000  1.00000 */ 
 sphere{<-3.71440, -8.65655,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.00000  1.00000  0.00000 */ 
 sphere{<-3.71440,  8.65655, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.00000  1.00000  1.00000 */ 
 sphere{<-3.71440,  8.65655,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  1.00000  0.00000  0.00000 */ 
 sphere{< 3.71440, -8.65655, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  1.00000  0.00000  1.00000 */ 
 sphere{< 3.71440, -8.65655,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  1.00000  1.00000  0.00000 */ 
 sphere{< 3.71440,  8.65655, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  1.00000  1.00000  1.00000 */ 
 sphere{< 3.71440,  8.65655,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.10800  0.47100  0.00000 */ 
 sphere{<-2.91209, -0.50208, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.10800  0.47100  1.00000 */ 
 sphere{<-2.91209, -0.50208,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.89200  0.52900  1.00000 */ 
 sphere{< 2.91209,  0.50208,  4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
 /* Sphere for Dy_  at  0.89200  0.52900  0.00000 */ 
 sphere{< 2.91209,  0.50208, -4.03480>,  0.40000
  texture{pigment{color Magenta  }}
   finish{phong  0.20 phong_size  1.00}
 }
/* Labels */
  text { ttf "crystal.ttf","o" 0.15,0
    scale <1.13,1.13,1.13>
    rotate <0, 0, -zrot>
    rotate <0, -yrot, 0>
    rotate <-xrot,0, 0>
    translate <-4.21213,-9.15863,-4.53512> pigment{color Black}
  }
/* Labels */
  text { ttf "crystal.ttf","a" 0.15,0
    scale <1.13,1.13,1.13>
    rotate <0, 0, -zrot>
    rotate <0, -yrot, 0>
    rotate <-xrot,0, 0>
    translate < 4.21213,-8.65655,-4.03480> pigment{color Black}
  }
/* Labels */
  text { ttf "crystal.ttf","b" 0.15,0
    scale <1.13,1.13,1.13>
    rotate <0, 0, -zrot>
    rotate <0, -yrot, 0>
    rotate <-xrot,0, 0>
    translate <-3.71440, 9.15863,-4.03480> pigment{color Black}
  }
/* Labels */
  text { ttf "crystal.ttf","c" 0.15,0
    scale <1.13,1.13,1.13>
    rotate <0, 0, -zrot>
    rotate <0, -yrot, 0>
    rotate <-xrot,0, 0>
    translate <-3.71440,-8.65655, 4.53512> pigment{color Black}
  }
 }
rotate <xrot, 0, 0>
rotate <0, yrot, 0>
rotate <0, 0, zrot>
translate <xmove, ymove, 0>
}
