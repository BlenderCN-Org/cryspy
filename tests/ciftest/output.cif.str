titl imported
import cif output.cif
box 0.02 Black
background White
view 0. 0. 0.
pack  -0.05 1.05 -0.05 1.05 -0.05 1.05
end
