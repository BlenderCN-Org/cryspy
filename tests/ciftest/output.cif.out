

                DRAWxtl
  Crystal Structure Drawing Program V5.5

  Copyright 1996, 1997, 2000-2011 by Larry Finger, Martin Kroeker, and Brian Toby
  (all rights reserved).

     Combined POVxtl and VRMLxtl
POVxtl  - Scenes for Persistence of Vision Ray Tracer (POV-Ray) V2.0
VRMLxtl - Virtual Reality Modeling Language V1.0/V2.0

 doing frame 1 of 1

********************* Input: ************************
*         titl imported
*         import cif output.cif 1
======================= Start of import/inline commands
******      Cell      7.42880 17.31310  8.06960   90.000   90.000   90.000
******  spgr P 1
******      Atom Dy_  11  0.70700  0.32800  0.05200
******      Atom Dy_  12  0.98200  0.08200  1.00000
******      Atom Dy_  13  0.01800  0.91800  3.00000
******      Atom Dy_  14  0.29300  0.67200  0.94800
******      Atom Dy_  15  1.00000  0.00000  0.00000
******      Atom Dy_  16  0.10800  0.47100  1.00000
******      Atom Dy_  17  0.89200  0.52900  3.00000
******      Atom Mn_  11  0.70700  0.32800  0.05200
******      Atom Mn_  12  0.98200  0.08200  1.00000
******      Atom Mn_  13  0.01800  0.91800  3.00000
******      Atom Mn_  14  0.29300  0.67200  0.94800
******      Atom Mn_  15  1.00000  0.00000  0.00000
******      Atom Mn_  16  0.10800  0.47100  1.00000
******      Atom Mn_  17  0.89200  0.52900  3.00000
******      Atom O_   11  0.70700  0.32800  0.05200
******      Atom O_   12  0.98200  0.08200  1.00000
******      Atom O_   13  0.01800  0.91800  3.00000
******      Atom O_   14  0.29300  0.67200  0.94800
******      Atom O_   15  1.00000  0.00000  0.00000
******      Atom O_   16  0.10800  0.47100  1.00000
******      Atom O_   17  0.89200  0.52900  3.00000
******      Atom O_   21  0.70700  0.32800  0.05200
******      Atom O_   22  0.98200  0.08200  1.00000
******      Atom O_   23  0.01800  0.91800  3.00000
******      Atom O_   24  0.29300  0.67200  0.94800
******      Atom O_   25  1.00000  0.00000  0.00000
******      Atom O_   26  0.10800  0.47100  1.00000
******      Atom O_   27  0.89200  0.52900  3.00000
======================= End of import/inline commands
*         box  0.020 Black
*         background White
*         axislines   0.00 Gray20
*         finish   0.70   0.30   0.08   0.01
*         sphere Dy_  0.400 Magenta
*         labeltext -0.067 -0.029 -0.062 o
*         labeltext  1.067  0.000  0.000 a
*         labeltext  0.000  1.029  0.000 b
*         labeltext  0.000  0.000  1.062 c
*         pack  -0.05   1.05  -0.05   1.05  -0.05   1.05
*         view   14.3  -23.9  -36.1
*****************************************************

Triclinic - Acentric - P lattice with 1 Symmetry Operators

+x,+y,+z
Lattice metrics
   7.429   -0.000   -0.000
   0.000   17.313   -0.000
   0.000    0.000    8.070

Master atom list contains    66 atoms.


 Bond Distance Analysis

Distances from Dy_  11 at  0.70700 0.32800 0.05200    to

 Mn_  11 at   0.70700  0.32800  0.05200     0.000
 O_   11 at   0.70700  0.32800  0.05200     0.000

Distances from Dy_  12 at  0.98200 0.08200 0.00000    to

 Mn_  12 at   0.98200  0.08200  0.00000     0.000
 O_   12 at   0.98200  0.08200  0.00000     0.000
 Dy_  15 at   1.00000  0.00000  0.00000     1.426
 Mn_  15 at   1.00000  0.00000  0.00000     1.426
 O_   15 at   1.00000  0.00000  0.00000     1.426

Distances from Dy_  13 at  0.01800 0.91800 1.00000    to

 Mn_  13 at   0.01800  0.91800  1.00000     0.000
 O_   13 at   0.01800  0.91800  1.00000     0.000
 Dy_  15 at   0.00000  1.00000  1.00000     1.426
 Mn_  15 at   0.00000  1.00000  1.00000     1.426
 O_   15 at   0.00000  1.00000  1.00000     1.426

Distances from Dy_  14 at  0.29300 0.67200 0.94800    to

 Mn_  14 at   0.29300  0.67200  0.94800     0.000
 O_   14 at   0.29300  0.67200  0.94800     0.000

Distances from Dy_  15 at  0.00000 0.00000 0.00000    to

 Mn_  15 at   0.00000  0.00000  0.00000     0.000
 O_   15 at   0.00000  0.00000  0.00000     0.000
 Dy_  12 at  -0.01800  0.08200  0.00000     1.426
 Mn_  12 at  -0.01800  0.08200  0.00000     1.426
 O_   12 at  -0.01800  0.08200  0.00000     1.426

Distances from Dy_  16 at  0.10800 0.47100 0.00000    to

 Mn_  16 at   0.10800  0.47100  0.00000     0.000
 O_   16 at   0.10800  0.47100  0.00000     0.000

Distances from Dy_  17 at  0.89200 0.52900 1.00000    to

 Mn_  17 at   0.89200  0.52900  1.00000     0.000
 O_   17 at   0.89200  0.52900  1.00000     0.000

Distances from Mn_  11 at  0.70700 0.32800 0.05200    to

 Dy_  11 at   0.70700  0.32800  0.05200     0.000
 O_   11 at   0.70700  0.32800  0.05200     0.000

Distances from Mn_  12 at  0.98200 0.08200 0.00000    to

 Dy_  12 at   0.98200  0.08200  0.00000     0.000
 O_   12 at   0.98200  0.08200  0.00000     0.000
 Dy_  15 at   1.00000  0.00000  0.00000     1.426
 Mn_  15 at   1.00000  0.00000  0.00000     1.426
 O_   15 at   1.00000  0.00000  0.00000     1.426

Distances from Mn_  13 at  0.01800 0.91800 1.00000    to

 Dy_  13 at   0.01800  0.91800  1.00000     0.000
 O_   13 at   0.01800  0.91800  1.00000     0.000
 Dy_  15 at   0.00000  1.00000  1.00000     1.426
 Mn_  15 at   0.00000  1.00000  1.00000     1.426
 O_   15 at   0.00000  1.00000  1.00000     1.426

Distances from Mn_  14 at  0.29300 0.67200 0.94800    to

 Dy_  14 at   0.29300  0.67200  0.94800     0.000
 O_   14 at   0.29300  0.67200  0.94800     0.000

Distances from Mn_  15 at  0.00000 0.00000 0.00000    to

 Dy_  15 at   0.00000  0.00000  0.00000     0.000
 O_   15 at   0.00000  0.00000  0.00000     0.000
 Dy_  12 at  -0.01800  0.08200  0.00000     1.426
 Mn_  12 at  -0.01800  0.08200  0.00000     1.426
 O_   12 at  -0.01800  0.08200  0.00000     1.426

Distances from Mn_  16 at  0.10800 0.47100 0.00000    to

 Dy_  16 at   0.10800  0.47100  0.00000     0.000
 O_   16 at   0.10800  0.47100  0.00000     0.000

Distances from Mn_  17 at  0.89200 0.52900 1.00000    to

 Dy_  17 at   0.89200  0.52900  1.00000     0.000
 O_   17 at   0.89200  0.52900  1.00000     0.000

Distances from O_   11 at  0.70700 0.32800 0.05200    to

 Dy_  11 at   0.70700  0.32800  0.05200     0.000
 Mn_  11 at   0.70700  0.32800  0.05200     0.000

Distances from O_   12 at  0.98200 0.08200 0.00000    to

 Dy_  12 at   0.98200  0.08200  0.00000     0.000
 Mn_  12 at   0.98200  0.08200  0.00000     0.000
 Dy_  15 at   1.00000  0.00000  0.00000     1.426
 Mn_  15 at   1.00000  0.00000  0.00000     1.426
 O_   15 at   1.00000  0.00000  0.00000     1.426

Distances from O_   13 at  0.01800 0.91800 1.00000    to

 Dy_  13 at   0.01800  0.91800  1.00000     0.000
 Mn_  13 at   0.01800  0.91800  1.00000     0.000
 Dy_  15 at   0.00000  1.00000  1.00000     1.426
 Mn_  15 at   0.00000  1.00000  1.00000     1.426
 O_   15 at   0.00000  1.00000  1.00000     1.426

Distances from O_   14 at  0.29300 0.67200 0.94800    to

 Dy_  14 at   0.29300  0.67200  0.94800     0.000
 Mn_  14 at   0.29300  0.67200  0.94800     0.000

Distances from O_   15 at  0.00000 0.00000 0.00000    to

 Dy_  15 at   0.00000  0.00000  0.00000     0.000
 Mn_  15 at   0.00000  0.00000  0.00000     0.000
 Dy_  12 at  -0.01800  0.08200  0.00000     1.426
 Mn_  12 at  -0.01800  0.08200  0.00000     1.426
 O_   12 at  -0.01800  0.08200  0.00000     1.426

Distances from O_   16 at  0.10800 0.47100 0.00000    to

 Dy_  16 at   0.10800  0.47100  0.00000     0.000
 Mn_  16 at   0.10800  0.47100  0.00000     0.000

Distances from O_   17 at  0.89200 0.52900 1.00000    to

 Dy_  17 at   0.89200  0.52900  1.00000     0.000
 Mn_  17 at   0.89200  0.52900  1.00000     0.000

Distances from O_   21 at  0.70700 0.32800 0.05200    to

 Dy_  11 at   0.70700  0.32800  0.05200     0.000
 Mn_  11 at   0.70700  0.32800  0.05200     0.000
 O_   11 at   0.70700  0.32800  0.05200     0.000

Distances from O_   22 at  0.98200 0.08200 1.00000    to

 Dy_  12 at   0.98200  0.08200  1.00000     0.000
 Mn_  12 at   0.98200  0.08200  1.00000     0.000
 O_   12 at   0.98200  0.08200  1.00000     0.000
 Dy_  15 at   1.00000  0.00000  1.00000     1.426
 Mn_  15 at   1.00000  0.00000  1.00000     1.426
 O_   15 at   1.00000  0.00000  1.00000     1.426

Distances from O_   23 at  0.01800 0.91800 1.00000    to

 Dy_  13 at   0.01800  0.91800  1.00000     0.000
 Mn_  13 at   0.01800  0.91800  1.00000     0.000
 O_   13 at   0.01800  0.91800  1.00000     0.000
 Dy_  15 at   0.00000  1.00000  1.00000     1.426
 Mn_  15 at   0.00000  1.00000  1.00000     1.426
 O_   15 at   0.00000  1.00000  1.00000     1.426

Distances from O_   24 at  0.29300 0.67200 0.94800    to

 Dy_  14 at   0.29300  0.67200  0.94800     0.000
 Mn_  14 at   0.29300  0.67200  0.94800     0.000
 O_   14 at   0.29300  0.67200  0.94800     0.000

Distances from O_   25 at  1.00000 0.00000 0.00000    to

 Dy_  15 at   1.00000  0.00000  0.00000     0.000
 Mn_  15 at   1.00000  0.00000  0.00000     0.000
 O_   15 at   1.00000  0.00000  0.00000     0.000
 Dy_  12 at   0.98200  0.08200  0.00000     1.426
 Mn_  12 at   0.98200  0.08200  0.00000     1.426
 O_   12 at   0.98200  0.08200  0.00000     1.426

Distances from O_   26 at  0.10800 0.47100 1.00000    to

 Dy_  16 at   0.10800  0.47100  1.00000     0.000
 Mn_  16 at   0.10800  0.47100  1.00000     0.000
 O_   16 at   0.10800  0.47100  1.00000     0.000

Distances from O_   27 at  0.89200 0.52900 1.00000    to

 Dy_  17 at   0.89200  0.52900  1.00000     0.000
 Mn_  17 at   0.89200  0.52900  1.00000     0.000
 O_   17 at   0.89200  0.52900  1.00000     0.000


Generated   22 spheres.
Label[1] -0.067 -0.029 -0.062 o
Label[2] 1.067 0 0 a
Label[3] 0 1.029 0 b
Label[4] 0 0 1.062 c
Generated    4 labels.
End of File.
